schedule_info.py

```diff
@@ -0,0 +1,247 @@
+        pages = []  # "strony" embeda, kazda z nich odpowiada jednej audycji
+        if day.lower() == "short":  # szczegolny przypadek jesli uzytkownik chce skrocona ramowke
+            short_embed = discord.Embed(title="Dziś jeszcze gramy:")
+            for broadcast in resp_json[week_days[datetime.datetime.today().weekday()]]:  # tylko dla danego dnia tygodnia sprawdza co jest grane, datetime.datetime.today().weekday() zwraca dzien tygodnia jako numer
+                if int(broadcast["end_hour"][:2]) > datetime.datetime.today().hour:  # broadcast["end_hour"][:2] bierze pierwsze dwie cyfry z godziny podanej w formacie hh:mm:ss
+                    short_embed.add_field(name=broadcast["title"], value=f'{broadcast["start_hour"]} - {broadcast["end_hour"]}', inline=False)
+            await ctx.send(embed=short_embed)
+            return
+
+        if day.lower() not in week_days:  # sprawdza czy dzien tygodnia podany przez uzytkownika jest poprawny
+            await ctx.channel.send("Wpisz poprawny dzień tygodnia w języku angielskim!")
+            return
+
+        for broadcast in resp_json[day]:  # dla kazdej audycji tworzy jedna "strone" embeda
+            embed = discord.Embed(title=broadcast["title"])
+            embed.add_field(name="Nazwa audycji:", value=broadcast["title"], inline=False)
+            embed.add_field(name="Od - do:", value=f'{broadcast["start_hour"]} - {broadcast["end_hour"]}', inline=False)
+            if broadcast["description"] is not None:
+                embed.add_field(name="Opis:", value=broadcast["description"], inline=False)
+            embed.set_image(url=broadcast["player_image"])
+            pages.append(embed)
+
+        message = await ctx.channel.send(embed=pages[0])    # wysyla pierwsza strone embeda
+        await message.add_reaction('⏮')     #dodaje kolejno reakcje pod embedem sluzace do przesuwania stron
+        await message.add_reaction('◀')
+        await message.add_reaction('▶')
+        await message.add_reaction('⏭')
+
+        def check(reaction, user):      # funkcja sprawdzajaca czy reakcja dodana pod embedem jest od autora komendy
+            return user == ctx.author
+
+        i = 0
+        reaction = None
+        while True:
+            if str(reaction) == '⏮':    #reakcja do powrotu do pierwszej strony emebda
+                i = 0
+                await message.edit(embed=pages[i])
+            elif str(reaction) == '◀':  #reakcja cofa embed o jedna strone
+                if i > 0:
+                    i -= 1
+                    await message.edit(embed=pages[i])
+            elif str(reaction) == '▶':  #reakcja przenosi na kolejna strone mebeda
+                if i < len(pages) - 1:
+                    i += 1
+                    await message.edit(embed=pages[i])
+            elif str(reaction) == '⏭':  #reakcja przenosi do ostatniej strony embeda
+                i = len(pages) - 1
+                await message.edit(embed=pages[i])
+            try:

```

Comment:
```suggestion
                await message.edit(embed=pages[i])
                
            try:
```
Żeby oddzielić dwa logiczne bloki robiące co innego

https://dev.to/suckup_de/do-not-fear-the-white-space-in-your-code-39d0